packages/transactions/src/coders/receipt.ts

```diff
@@ -774,6 +774,18 @@ export const getMintedAssetId = (contractId: string, subId: string): string => {
   return sha256(concat([contractIdBytes, subIdBytes]));
 };
 
+type AssetIdInput = {
```

Comment:
We already have a [type](https://github.com/FuelLabs/fuels-ts/blob/1fb529bf03e5751e61afac69bfbd488e05577d10/packages/interfaces/src/index.ts#L33-L35) for an `AssetId`, please remove this and use that instead.

Reply:
Thank you, I will use the one from fuel-ts/interfaces instead